pkg/snmp/traps/oid_resolver.go

```diff
@@ -163,12 +183,45 @@ func (or *MultiFilesOIDResolver) updateFromReader(reader io.Reader, unmarshalMet
 
 func (or *MultiFilesOIDResolver) updateResolverWithData(trapDB trapDBFileContent) {
 	definedVariables := variableSpec{}
-	for variableOID, variableData := range trapDB.Variables {
-		variableOID := NormalizeOID(variableOID)
+
+	allOIDs := make([]string, 0, len(trapDB.Variables))
+	for variableOID := range trapDB.Variables {
+		if !IsValidOID(variableOID) {
+			log.Errorf("trap variable OID %s does not look like a valid OID", variableOID)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏A warning might be more appropriate ? since it doesn't break anything and the program will still continue running for valid cases.